NEWS.txt

```diff
@@ -12,6 +12,11 @@ We recommend that every developer updates to this version.
 
 Highlights:
  - Make sure MKL uses GNU OpenMP
+
+   - **NB**: Matrix dot product (``gemm``) with numpy ``1.13`` and ``mkl`` from conda
```

Comment:
I would do:
Matrix dot product (``gemm``) with numpy ``1.13`` and ``mkl`` from conda
->
Matrix dot product (``gemm``) with and ``mkl 2018`` from conda

I think this is independent of numpy and depend only on the mkl version. I just tried to install something and it tried to install numpy 1.13 with mkl 2017 in conda. 

@abergeron does that sound good to you?

Reply:
Willie Parish IPHONE 12 Pro max